frontend/src/metabase/components/FieldSet.tsx

```diff
@@ -20,6 +29,7 @@ export default function FieldSet({
           className="h5 text-bold text-uppercase px1 text-nowrap text-medium"
         >
           {legend}
+          {required && <span>&nbsp;*</span>}
```

Comment:
😄 afaik span cannot be moved to another line anyway, so just ` ` could work here

Reply:
I am used to agressive formatting, which puts spaces in `{" "}` so they'er not lost with line wrapping — hence the `&nbsp`. We don't have this formatting rules yet (and hopefully won't have ever), but I don't like the version in curlies and just a plain space doesn't communicate the intention well enough in my opinion.